UltraStar Play/Assets/Common/Model/Song/SongMeta.cs

```diff
@@ -76,6 +78,10 @@ Encoding encoding
         if (filename == null)
         {
             throw new ArgumentNullException("filename");
+        }
+		if (songHash == null)
+        {
```

Comment:
The indentation seems to use tabs instead of spaces sometimes. Please check that your editor always uses spaces for indentation.